CMakeLists.txt

```diff
@@ -169,6 +228,15 @@ if (NOT PICOTOOL_NO_LIBUSB)
     endif()
 endif()
 
+if (TARGET mbedtls)
+    add_custom_target(binary_data_no_libusb DEPENDS
```

Comment:
`binary_data` is confusing because it has meaning in SDK; maybe something with "embedded"